content/04. box_1_experimental_design.md

```diff
@@ -0,0 +1,40 @@
+#### Box 1: Understanding experimental design of machine learning to inform requirements for data
+
+##### Components of ML experiments
+
+Machine learning algorithms identify patterns that explain or fit a given dataset.
+Every machine learning algorithm (_model_) goes through a _training_ phase where it identifies underlying patterns in a given dataset, and a _testing_ phase where the model applies the identified patterns to unseen data points.
```

Comment:
```suggestion
Every machine learning algorithm goes through a _training_ phase where it identifies underlying patterns in a given dataset to create a "trained" algorithm (a _model_) , and a _testing_ phase where the model applies the identified patterns to unseen data points.
```

I'd consider the "model" to be the trained predictor, not the naive algorithm, what do you all think?